test/suites/ekuiper/config_test.go

```diff
@@ -0,0 +1,75 @@
+package test
+
+import (
+	"edgex-snap-testing/test/utils"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestConfigOption(t *testing.T) {
+	const (
+		defaultPort = "6379"
+		newPort     = "6000"
+		portKey     = "config.edgex.default.port"
+	)
+
+	t.Cleanup(func() {
+		utils.SnapUnset(t, ekuiperSnap, portKey)
+		utils.SnapRestart(t, ekuiperSnap)
```

Comment:
Shouldn't the cleanup stop the service since the tests attempt to start it?

Reply:
The reason I add SnapRestart is to pick up the changes from SnapUnset. I added a SnapStop here.

Reply:
The following start in another test will make that happen. Restarting just before stopping isn't needed.